src/app/components/student/student.component.scss

```diff
@@ -1,25 +1,20 @@
+$silver-color: #e6e6e6;
+$text-center: center;
+$text-color: #131212;
```

Comment:
Please specify what is this color, like: `$text-color-gray` or `$text-color-lightblue`